kilkari/src/main/java/org/motechproject/nms/kilkari/domain/Subscriber.java

```diff
@@ -89,6 +89,21 @@ public void setLanguage(Language language) {
         this.language = language;
     }
 
+    @Ignore
+    public Set<Subscription> getAllSubscriptions() {
+        // TODO: I have no idea why I need to do this, but returning just this.subscriptions always results in
```

Comment:
Can you put a ticket in, or ask about this on the mailing list?  You can also try to repro it in motech/platform/mds/mds-test-bundle